survey/dashboards/config.py

```diff
@@ -0,0 +1,124 @@
+import os
+import json
+from django.conf import settings
+
+
+SORT_CONFIG_PATH = os.path.join(
+    settings.BASE_DIR, "data", "survey_config", "sort_only_config.json"
+)
+DEMOGRAPHIC_CONFIG_PATH = os.path.join(
+    settings.BASE_DIR, "data", "survey_config", "demography_only_config.json"
```

Comment:
Should be based on either survey.suvey_config or survey.demography_config instead? As user may change the configuration.